### ISSUE TITLE
Provide SQLAlchemy async demo

### ISSUE THREAD

**Issue by hongbo-miao** [trinodb/trino-python-client#408]
### Describe the feature

It would be great to provide a SQLAlchemy async demo, thanks!

SQLAlchemy says it supports async since version 1.4

This repo readme says

> trino.sqlalchemy is compatible with the latest 1.3.x and 1.4.x SQLAlchemy versions at the time of release of a particular version of the client.

https://docs.sqlalchemy.org/en/20/orm/extensions/asyncio.html

However, I cannot make them work.

I guess for SQLAlchemy, I need install

```shell
pip install sqlalchemy[asyncio]
```

This repo I need run

```shell
pip install trino[sqlalchemy]
```

Apparently, this does not exist 🥲

```shell
pip install trino[sqlalchemy[asyncio]]
```

I guess it may be blocked by this https://github.com/trinodb/trino-python-client/issues/3 (?)

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by hovaesco** [comment]
Async SQLAlchemy is not supported currently for trino-python-client.

**Comment by hongbo-miao** [comment]
Thanks @hovaesco for clarification! I guess it would be a feature request. Just updated the title. Hopefully one day can support. 😃

**Comment by mvanderlee** [comment]
@hongbo-miao you can give aiotrino 0.3.0 a try. I've brought it to feature parity with trino-python-client 0.333.0 and now includes sqlalchemy support.

**Comment by lozbrown** [comment]
> @hongbo-miao you can give aiotrino 0.3.0 a try. I've brought it to feature parity with trino-python-client 0.333.0 and now includes sqlalchemy support. 

Including spooling?

**Comment by mvanderlee** [comment]
> > [@hongbo-miao](https://github.com/hongbo-miao) you can give aiotrino 0.3.0 a try. I've brought it to feature parity with trino-python-client 0.333.0 and now includes sqlalchemy support.
> 
> Including spooling?

Yes. I've copied the test suite as well and modified it for asyncio. It all passed. Beyond that I haven't done much sqlalchemy and spooling testing due to time constraints.

**Comment by lozbrown** [comment]
Nice, we use aiotrino for a few things so nice to know it's up to parity
